ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/sno/cucushift-installer-rehearse-aws-ipi-sno-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-sno
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: single-node-conf-aws
+    - chain: ipi-install
+    test:
```

Comment:
we don't write `test` section in installer workflow, this will be updated in job configuration.

Reply:
OK, I get it. But I don't know how to update the test section to the job configuration, could you pls tell me?